templates/protobuf.tpl

```diff
@@ -0,0 +1,134 @@
+{# Protobuf template #}
+{# (C) 2022 Robert Bosch GmbH #}
+// protobuf code generated by protobuf.tpl
+{# https://developers.google.com/protocol-buffers/docs/proto#services #}
+syntax = "proto3";
+
+{# Define Types #}
+{# Not all vsc types can be represented as is in protobuf  #}
+{% set typedefs = dict() %}
+{# Add all type conversion  #}
+{% set x=typedefs.__setitem__("int16", "int32") %}
+{% set x=typedefs.__setitem__("uint8", "uint32") %}
+{% set x=typedefs.__setitem__("boolean", "bool") %}
+
```

Comment:
OK for now, but see comment about type conversions/warning.  In some cases, I think it might be better to refactor this code of the template into a custom python function.